CDP4Composition/LayoutGroupNames.cs

```diff
@@ -0,0 +1,29 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="RegionNames.cs" company="RHEA System S.A.">
```

Comment:
If you find a non AGPL license header, please change it to the correct AGPL version.